### PR TITLE
src/Portage/EBuild.hs: Remove obsolete 'src_uri' function

### PR DESCRIPTION
**Pull request by hololeap**
This removes the generation of `SRC_URI` from ebuilds generated by `hackport`.

A default `SRC_URI` pointing to [hackage.haskell.org](https://hackage.haskell.org) is now handled automatically by `haskell-cabal.eclass` and does not normally need to be added explicitly to the ebuild.

See: https://github.com/gentoo-haskell/gentoo-haskell/commit/ea245047d18f82e05656f1664c12a53eaf9afd57

### CODE REVIEW

**Review by solpeth — APPROVED** [review]

**Comment by hololeap** [comment]
@solpeth 
I want to make sure the second commit works as intended, then I'll merge it. Feel free to test, if you can.

**Comment by solpeth** [comment]
No worries, I'll give it a run.

File: src/Portage/EBuild.hs [hunk 1]
```diff
@@ -118,17 +102,15 @@ showEBuild now ebuild =
...
+     Just pn -> ss "CABAL_PN=". quote pn. nl.).
+  nl.
   ss "CABAL_FEATURES=". quote' (sepBy " " $ map render (features ebuild)). nl.
   ss "inherit haskell-cabal". nl.
   nl.
-  (case my_pn ebuild of
-     Nothing -> id
-     Just pn -> ss "MY_PN=". quote pn. nl.
-                ss "MY_P=". quote "${MY_PN}-${PV}". nl.
-                ss "S=". quote ("${WORKDIR}/${MY_P}"). nl. nl).
```
Comment (solpeth) [inline src/Portage/EBuild.hs:128]: As per build failure: can we remove these outermost parentheses? I have a feeling they may be redundant.

File: src/Portage/EBuild.hs [hunk 2]
```diff
@@ -118,17 +102,15 @@ showEBuild now ebuild =
   ss ("# ebuild generated by hackport " ++ hackportVersion ebuild). nl.
   sconcat (map (\(k, v) -> ss "#hackport: " . ss k . ss ": " . ss v . nl) $ used_options ebuild).
   nl.
+  (case cabal_pn ebuild of
+     Nothing -> id
+     Just pn -> ss "CABAL_PN=". quote pn. nl.).
```
Comment (solpeth) [inline src/Portage/EBuild.hs:107]: Same here, and then remove the second function composition operator `(.)`. That should hopefully work.

**Review by solpeth — CHANGES_REQUESTED** [review]

File: src/Portage/EBuild.hs (hunk 2 above)
Comment (hololeap) [reply src/Portage/EBuild.hs]: I think I got it figured out. That module is admittedly pretty confusing syntactically.

**Pull request merged**